homeassistant/auth/providers/homeassistant.py

```diff
@@ -257,7 +257,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     async def async_user_meta_for_credentials(
             self, credentials: Credentials) -> UserMeta:
         """Get extra info for this credential."""
-        return UserMeta(name=credentials.data['username'], is_active=True)
+        return UserMeta(name=credentials.data['username'])
```

Comment:
I think that this line shows exactly why this is probably not a good idea.

Now using the `homeassistant` provider will override the name on each login? That seems not logical.

Reply:
It will only overwrite it when the configured name is not ``None``. However, that will probably never be the case in case of the homeassistant auth provider, because an empty name is represented as the empty string there, I think.